code/modules/spells/spell_types/shadow_walk.dm

```diff
@@ -1,3 +1,5 @@
+#define SHADOW_REGEN_RATE 1.5
```

Comment:
Is this a buff over the original (1, 1, 0, BODYPART_ORGANIC)?

Reply:
The old version was inconsistent and they would heal multiple times depending on circumstances. This should consistently heal them for SHADOW_REGEN_RATE * 2 = 3 damage per second.